src/components/POLine/POLine.js

```diff
@@ -2,7 +2,9 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { get } from 'lodash';
 
-import { ORDER } from '../Utils/resources';
+import {
+  ORDER,
```

Comment:
any reason to make this change?

Reply:
It's happened during resolving conflicts. Fixed